packages/api/amplify_api_dart/lib/src/graphql/factories/graphql_request_factory.dart

```diff
@@ -251,6 +256,20 @@ class GraphQLRequestFactory {
     };
   }
 
+  Map<String, dynamic> buildVariablesForSubscriptionRequest({
+    required ModelType modelType,
+    QueryPredicate? where,
+  }) {
+    if (where == null) {
+      return {};
+    }
+    final filter = GraphQLRequestFactory.instance
```

Comment:
question: Will this work for a subscription w owner field? I wasn't sure if it would implicitly work from the filter or if the owner value had to be moved into another variable value as I thought those subscriptions then took 2 variables (filter and owner).

Reply:
discussed offline, sounds like no, which is ok IMO for this PR. Can be separate effort.